Daq 218 epics marshaller

MattTaylorDLS opened the pull request:
Just a look at the tests that were copied from existing ones that use the json marshaller. More will need to be created to cover off all models and regions, but until the structures are finalised, I didn't want to spend too much time on them to save on rework later.

Pay particular attention to how the Jars are added to the bundle, and used, as I'm not sure this is what we want.

Commit:
Added tests for serialisation of compound model using the EPICS V4
Marshaller.

Commit:
Rename packages for EPICS serialisation tests

gerring committed:
Merge branch 'master' into daq-218EpicsMarshaller

gerring commented:
I think your commit does not reference a jira ticket. You must paste the ticket number into the commit message.

Commit:
EPICS V4 serialisation changes.

Added bundle for epics connector service.
Changed tests to use that bundle, and fixed tests after modifications to
the Bounding Box changed the output of serialisation.
http://jira.diamond.ac.uk/browse/DAQ-218

Commit:
Update pom.xml

Added connector.epics.
http://jira.diamond.ac.uk/browse/DAQ-218

Commit:
Bring connector.epics pom.xml into line with other bundles

http://jira.diamond.ac.uk/browse/DAQ-218

Commit:
Merge branch 'daq-218EpicsMarshaller' of https://github.com/DiamondLightSource/daq-eclipse.git

gerring committed:
Merge branch 'master' into daq-218EpicsMarshaller
